doc/for-test-authors.rst

```diff
@@ -392,6 +392,25 @@ This is actually a convenience function that combines two other matchers:
 Raises_ and MatchesException_.
 
 
+The is_deprecated helper
```

Comment:
I suggest changing the name - 'is_deprecated' is too specific and implies something about the warning message that we don't know.